packages/geo/src/lib/query/shared/query.service.ts

```diff
@@ -64,23 +64,22 @@ export class QueryService {
       });
     }
 
-    const newLayers = layers
-      .filter((layer: Layer) => layer.visible && layer.isInResolutionsRange)
-      .map((layer: Layer) => this.queryLayer(layer, options));
+    const queries$ = layers
+      .filter((layer) => layer.displayed === true)
+      .map((layer) => this.queryLayer(layer, options));
     // the directive accept array in this format [observable, observable...]
     // if we use multiple 'url' in queryUrl so the result => this form [observable, observable, [observable, observable]]
     // so we need to flat the array
-    // eslint-disable-next-line prefer-spread
-    const flatArray = [].concat.apply([], newLayers);
-    return flatArray;
+    const flattenedQueries$ = [].concat.apply([], queries$);
```

Comment:
Use the spread operator instead of '.apply()'.eslintprefer-spread

Reply:
On pourrait créer un ticket à part pour change ce comportement. Je n'ai pas le temps nécessaire présentement pour me relancer dans des tests sur d'autres changements.